File: data/java/FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<String> param0 = new ArrayList<>();
    param0.add("7");
    param0.add("305745689");
    param0.add("444");
    param0.add("4");
    param0.add("2074");
    param0.add("27");
    param0.add("447");
    param0.add("255");
    param0.add("10000111111011");
    param0.add("fAKcSDRTNz");
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}

File: data/python/FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7_MAIN.py
if __name__ == '__main__':
    param = [
        ('7',),
        ('305745689',),
        ('444',),
        ('4',),
        ('2074',),
        ('27',),
        ('447',),
        ('255',),
        ('10000111111011',),
        ('fAKcSDRTNz',)
    ]
    n_success = 0
    print("class,mutant,test_index,result\n")
    for i, parameters_set in enumerate(param):
        try:
            print("FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("FIND_POSITION_GIVEN_NUMBER_AMONG_NUMBERS_MADE_4_7," + sys.argv[1] + "," + str(i) + ",EXCEPTION")
